packages/ember-glimmer/tests/integration/helpers/get-test.js

```diff
@@ -0,0 +1,224 @@
+      possibleValues: {
+        colors1: { apple: 'red', banana: 'yellow' },
+        colors2: { apple: 'green', banana: 'purple' }
+      },
+      objectKey: 'colors1',
+      key: 'apple'
+    });
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[green] [green]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'key', 'banana'));
+    this.assertText('[yellow] [yellow]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[purple] [purple]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[yellow] [yellow]');
+  }
+
+  ['@test should be able to get an object value with a GetStream value and GetStream key']() {
+    this.render(`[{{get (get possibleValues objectKey) (get possibleKeys key)}}] [{{if true (get (get possibleValues objectKey) (get possibleKeys key))}}]`, {
+      possibleValues: {
+        colors1: { apple: 'red', banana: 'yellow' },
+        colors2: { apple: 'green', banana: 'purple' }
+      },
+      objectKey: 'colors1',
+      possibleKeys: {
+        key1: 'apple',
+        key2: 'banana'
+      },
+      key: 'key1'
+    });
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[green] [green]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'key', 'key2'));
+    this.assertText('[yellow] [yellow]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[purple] [purple]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[yellow] [yellow]');
+  }
+
+  ['@test should handle object values as nulls']() {
+    this.render(`[{{get colors 'apple'}}] [{{if true (get colors 'apple')}}]`, {
+      colors: null
+    });
+    this.assertText('[] []');
+
+    this.runTask(() => set(this.context, 'colors', { apple: 'green', banana: 'purple' }));
+    this.assertText('[green] [green]');
+
+    this.runTask(() => set(this.context, 'colors', null));
+    this.assertText('[] []');
+  };
+
+  ['@test should handle object values and keys as nulls']() {
+    this.render(`[{{get colors 'apple'}}] [{{if true (get colors key)}}]`, {
+      colors: null,
+      key: null
+    });
+    this.assertText('[] []');
+  };
+
+  // Looks like some misconfiguration here.
+  //
+  // Htmlbars: Error: Assertion Failed: HTMLBars error: Could not find component named "-text-field" (no component or template with that name was found)
+  // Glimmer: Compile Error: input is not a helper
+  ['@skip get helper value should be updatable using {{input}} and (mut) - dynamic key']() {
```

Comment:
I get exceptions dumped in comments above for both `[htmlbars]` and `[glimmer]` modes.
`[glimmer]` fault is clear enough. Currently can't figure out why does `[htmlbars]` mode fails while the old-fashioned htmlbars tests work. Looks like some kind of `RenderingTest` test misconfiguration. Haven't investigated yet.